app/api/v1/mediator/tally.py

```diff
@@ -86,6 +86,11 @@ def tally_ballots(
     # get the cast and spoiled ballots by checking the current ballot inventory
     # and filtering the table for
     inventory = get_ballot_inventory(election_id, request.app.state.settings)
+    if inventory is None:
+        raise HTTPException(
```

Comment:
This is just a copy-paste of the throw code that used to occur in get_ballot_inventory, I just moved it up a tier